src/serialbox-python/setup.py

```diff
@@ -0,0 +1,145 @@
+# -*- coding: utf-8 -*-
+import os
+import re
+import subprocess
+import sys
+
+import os
+from setuptools import setup, Extension, find_packages
+from setuptools.command.build_ext import build_ext
+
+DIR = os.path.abspath(os.path.dirname(__file__))
+
+# Convert distutils Windows platform specifiers to CMake -A arguments
```

Comment:
Copied from the cmake setup.py template. No idea whether it does or does not compile on windows. I'd be happy to remove the windows stuff if you can point out what is safe/appropriate to remove, though the only issue it might cause is someone thinking it works on Windows if it does not.